redis-wrapper/src/main/java/redis/RedisCluster.java

```diff
@@ -231,6 +232,17 @@ public List<String> getItems(List<String> keys) throws Exception {
         }
     }
 
+    @Override
+    public HashSet<String> getMembers(String key) throws Exception {
+        Set<String> flags;
```

Comment:
Maybe a more generic name would be more suitable?
```suggestion
        Set<String> items;
```